src/ahbicht/evaluation_results.py

```diff
@@ -77,7 +81,7 @@ def deserialize(self, data, **kwargs) -> FormatConstraintEvaluationResult:
 
 
 @attr.s(auto_attribs=True, kw_only=True)
-class ConditionCheckResult:
+class AhbExpressionEvaluationResult:
     """
     A class for the result of the overall condition check.
```

Comment:
vllt könntest du hier "overall" noch spezifizieren? wir haben ja festgestellt, dass das mit einigem abstand nicht mehr 100% intuitiv ist.

Reply:
Ah, guter Punkt, das war ja der Hauptgrund für die Umbenennung :D

Reply:
https://github.com/Hochfrequenz/ahbicht/pull/83/commits/897edeab1dcb53604f5e8f1e2fab72b6ce70738e

Reply:
können wir es *content* evaluation nennen?

Reply:
Also es ist das was rauskommt, wenn man den parsedTree von der ahb_expression in den AhbTransformer steckt.
Hätte gesagt, dass das nicht nur content evaluation, sondern auch expression evaluation ist.